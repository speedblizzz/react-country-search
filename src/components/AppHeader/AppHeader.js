import React from 'react';
import { Translate } from 'react-redux-i18n';

import './AppHeader.scss';

const AppHeader = () => {
  return (
    <h1 className="app-title">
      <Translate value="application.appTitle" />
    </h1>
  );
};

export default AppHeader;
