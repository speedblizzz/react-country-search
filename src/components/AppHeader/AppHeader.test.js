import React from 'react';
import { render } from '@testing-library/react';

import AppHeader from './AppHeader';

test('should ', () => {
  const { container } = render(<AppHeader />);
  expect(container).toMatchSnapshot();
});
