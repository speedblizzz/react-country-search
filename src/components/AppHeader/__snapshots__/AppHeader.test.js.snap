// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should  1`] = `
<div>
  <h1
    class="app-title"
  >
    <span>
      appTitle
    </span>
  </h1>
</div>
`;
